client/src/pages/course/mentor/confirm.tsx

```diff
@@ -31,21 +33,22 @@ function Page(props: { session: Session; courseAlias?: string }) {
   useAsync(async () => {
     try {
       setLoading(true);
-      const courseAlias = router.query?.course;
-      if (!courseAlias) {
+      const courseAlias = router.query?.course as string;
+      if (courseAlias == null) {
         return;
       }
       const courses = await new CoursesService().getCourses();
-      const course = courses.find(c => c.alias === courseAlias) ?? null;
+      const course = courses.find(c => c.alias.toLowerCase() === courseAlias.toLowerCase()) ?? null;
```

Comment:
Are they stored not in lower case?

Reply:
And plese see the comment above

Reply:
no, they are stored in different registers